docs/cli/fleet-cli/fleet_apply.md

```diff
@@ -46,6 +46,11 @@ fleet apply [flags] BUNDLE_NAME PATH...
       --target-namespace string                Ensure this bundle goes to this target namespace
       --targets-file string                    Addition source of targets and restrictions to be append
       --username string                        Basic auth username for helm repo
+      --driven-scan                            Use driven scan. Bundles are defined by the user.
+                                               Paths will be defined by the tuple <base, options> separated by a separator.
+                                               The default separator is the character :. If you need to use a different one, 
+                                               use the --driven-scan-sep flag.
+      --driven-scan-sep                        Separator used for user-driven bundle generation paths. (default: :)
 ```
 
```

Comment:
This makes me think that we should find a way to automatically update this repo when updating flag descriptions in our CLI, to make sure that both are in sync :thinking: